### PR TITLE
added display_title to publication interactions

### PR DESCRIPTION
**Pull request by matt-borgelt**
Requires the modification to view pubs.all_publication_lookup to function properly, this is in gitlab

### CODE REVIEW

File: src/main/resources/xslt/jats-preview-xslt/xslt/main/jats-html.xsl [hunk 1]
```diff
@@ -2250,12 +2250,7 @@ or pipeline) parameterized.
   </xsl:template>
   
 
-  <xsl:template priority="2" mode="list"
-    match="list[@list-type='simple' or list-item/label]">
-    <ul style="list-style-type: none">
-      <xsl:apply-templates select="list-item"/>
-    </ul>
-  </xsl:template>
+
```
Comment (orosz-usgs) [inline src/main/resources/xslt/jats-preview-xslt/xslt/main/jats-html.xsl:2253]: my knowledge of xslt and the html here is lacking. As long as the html output has been reviewed,
it's good.

File: src/main/resources/xslt/jats-preview-xslt/xslt/main/jats-html.xsl [hunk 2]
```diff
@@ -3238,6 +3233,7 @@ or pipeline) parameterized.
       Pass in the desired label if a label is to be autogenerated  
       <xsl:with-param name="auto-text">
         <xsl:text>Appendix </xsl:text>
+
```
Comment (orosz-usgs) [inline src/main/resources/xslt/jats-preview-xslt/xslt/main/jats-html.xsl]: minor formatting/style, extra blank line

File: src/test/java/gov/usgs/cida/pubs/dao/LinkTypeDaoIT.java [hunk 3]
```diff
@@ -22,7 +22,7 @@
 	classes={DbTestConfig.class, LinkTypeDao.class})
 public class LinkTypeDaoIT extends BaseIT {
 
-	public static final int LINK_TYPES_CNT = 33;
+	public static final int LINK_TYPES_CNT = 38;
```
Comment (orosz-usgs) [inline src/test/java/gov/usgs/cida/pubs/dao/LinkTypeDaoIT.java:25]: I probably  missed this, I do not see where the link type count would have increase from
33 to 38.

Comment (matt-borgelt) [reply src/test/java/gov/usgs/cida/pubs/dao/LinkTypeDaoIT.java:25]: There was an earlier commit in pubswh-ui that increased the number

**Pull request merged**